bench/lib/set_data_perf.js

```diff
@@ -7,7 +7,9 @@ module.exports = function(source, numCalls, geojson, cb) {
     var startTime = null;
     var times = [];
 
-    source.on('tile.load', function tileCounter() {
+    source.on('data', function tileCounter(event) {
+        if (event.dataType !== 'tile') return;
```

Comment:
Does this event binding need to distinguish between "tile loading" and "tile unloading"?